### ISSUE TITLE
Exported properties are not hot reloaded in NativeScript

### ISSUE THREAD

**Issue by metsawyr** [godotengine/godot#37495]
**Godot version:**
Godot v3.2.1

**OS/device including version:**
macOS Mojave 10.14.6

**Issue description:**
Interface of node doesn't get hot reloaded on dylib rebuild. By that I mean if I have new properties exported (or old ones deleted) I won't see updates on the gui of Godot until I relaunch Godot or exit to project manager and reload the project.

**Steps to reproduce:**
```
Requires rust to be installed (1.42.0)
1. `$ cargo build`
2. open the project in Godot
3. copy/paste line 7, 8 and 15 in src/lib.rs renaming the new property
4. `$ cargo build`
5. observe interface haven't been updated
6. exit to project manager, load the project again and ensure that property exists
```

**Minimal reproduction project:**
[InterfaceBug.zip](https://github.com/godotengine/godot/files/4415762/InterfaceBug.zip)

**Comment by Calinou** [comment]
> 6. exit to project manager, load the project again and ensure that property exists

You can use **Scene > Revert Scene** after saving the scene to avoid restarting the whole editor.

**Comment by Bromeon** [comment]
In GDExtension, this has been implemented for Godot 4.2 in https://github.com/godotengine/godot/pull/80284.
I guess this can be closed (also since GDNative will probably not see major changes anymore)?
